clients/githubrepo/tarball.go

```diff
@@ -77,6 +77,7 @@ func (handler *tarballHandler) init(ctx context.Context, repo *github.Repository
 
 	// Setup temp dir/files and download repo tarball.
 	if err := handler.getTarball(ctx, repo); errors.Is(err, errTarballNotFound) {
+		log.Printf("unable to get tarball %v. Skipping...", err)
```

Comment:
It is, but its a server-side error. We can't really do much on our side. So instead of failing, choosing to ignore it instead.

Reply:
Would retrying help?

Reply:
Discussed offline. Retrying doesn't help here since the downloads don't resume and instead start from scratch. Also, this currently happens on 2 repos - https://github.com/cdnjs/cdnjs, https://github.com/echen102/COVID-19-TweetIDs. Both of these don't contain code, so ok to to skip.